diff --git a/src/lib/lua.cc b/src/lib/lua_rime.cc
similarity index 99%
rename from src/lib/lua.cc
rename to src/lib/lua_rime.cc
index fab99bf..5470921 100644
--- a/src/lib/lua.cc
+++ b/src/lib/lua_rime.cc
@@ -1,4 +1,4 @@
-#include "lua.h"
+#include "lua_rime.h"
 #include "lua_templates.h"
 
 namespace LuaImpl {
diff --git a/src/lib/lua.h b/src/lib/lua_rime.h
similarity index 100%
rename from src/lib/lua.h
rename to src/lib/lua_rime.h
diff --git a/src/lib/lua_templates.h b/src/lib/lua_templates.h
index a98a577..77590c3 100644
--- a/src/lib/lua_templates.h
+++ b/src/lib/lua_templates.h
@@ -1,7 +1,7 @@
 #ifndef LIB_LUA_TEMPLATES_H_
 #define LIB_LUA_TEMPLATES_H_
 
-#include "lua.h"
+#include "lua_rime.h"
 #include <typeinfo>
 #include <vector>
 #include <set>
diff --git a/src/lua_gears.h b/src/lua_gears.h
index d763970..9d6d18c 100644
--- a/src/lua_gears.h
+++ b/src/lua_gears.h
@@ -7,7 +7,7 @@
 #include <rime/segmentor.h>
 #include <rime/processor.h>
 #include <rime/gear/filter_commons.h>
-#include "lib/lua.h"
+#include "lib/lua_rime.h"
 
 namespace rime {
 
